packages/http/src/__tests__/__snapshots__/http-prism-instance.spec.ts.snap

```diff
@@ -26,39 +26,3 @@ Object {
   },
 }
 `;
-
```

Comment:
Removing reliance on snapshots for some of the most important tests because it is hard to read them.